icon credit

Btimes2 opened the pull request:
closes #43

Btimes2 committed:
icon credit

closes #43

Sasszem reviewed (changes requested):
Please remove trailing newlines!
Please also add a link to the original in `README.md`!

Btimes2 committed:
Merge branch 'master' of https://github.com/Sasszem/13 into issue43

Btimes2 committed:
deleting trailing line breaks
it is astonishing what some editors do behind your back
now this really closes #43

Btimes2 commented:
I linked the original icon in Credits.MD. 
I also accidentally pushed it onto a different branch, issue40.
So when you merge them, you will get the credit from that branch.

Sasszem reviewed (approved):
Perfect!
